node/src/main/kotlin/net/corda/node/services/vault/HibernateQueryCriteriaParser.kt

```diff
@@ -594,7 +581,37 @@ class HibernateQueryCriteriaParser(val contractStateType: Class<out ContractStat
             }
         }
 
-        return emptySet()
+        val predicateSet = mutableSetOf<Predicate>()
+
+        // Participants.
+        criteria.participants?.let {
+            val participants = criteria.participants!!
+            // Get the persistent party entity.
+            val persistentPartyEntity = VaultSchemaV1.PersistentParty::class.java
+            val entityRoot = rootEntities.getOrElse(persistentPartyEntity) {
+                val entityRoot = criteriaQuery.from(persistentPartyEntity)
+                rootEntities[persistentPartyEntity] = entityRoot
+                entityRoot
+            }
+            val predicateID = Pair(VaultSchemaV1.PersistentParty::x500Name.name, IN)
+            // Add a new predicate or update th existing one.
```

Comment:
```suggestion
            // Add a new predicate or update the existing one.
```
- Missing an 'e' on the 'the'.